### ISSUE TITLE
Can we provide a bidirectional LSTM from scratch?

### ISSUE THREAD

**Issue by liuzhi136** [zackchase/mxnet-the-straight-dope#214]
As title mentioned, could we provide a tutorial for bidirectional LSTM/RNN from scratch? I want to know the details for implementing the backward hidden states. @zackchase

**Comment by zackchase** [comment]
Yes, for sure. I'll add I to my to-do. Let's keep this issue open until we have a bidirectional lstm

**Comment by liuzhi136** [comment]
Thanks！
